src/components/pickers/TimePicker.js

```diff
@@ -171,16 +171,18 @@ export default {
       } else if (allowed instanceof Function) {
         return allowed(value)
       } else if (allowed instanceof Object) {
-        const range = type === 'minute' && this.ranges.minutes || !this.is24hr && this.ranges.hours.slice(1, 13) || this.ranges.hours
+        const range = type === 'minute' && this.ranges.minutes ||
+          !this.is24hr && this.ranges.hours.slice(1, 13) ||
```

Comment:
.slice returns a copy and doesn't actually modify the array. So I'm not sure what this line does.